### ISSUE TITLE
Unable to build on Linux armhf architecture (bad `%lld` arg)

### ISSUE THREAD

**Issue by jhonny-oliveira** [deskflow/deskflow#7458]
### Discussion

- [X] Yes, and I still want to report a bug

### What type of Synergy user are you?

Community user

### Product name

Synergy 1

### Synergy version number

1.15.1+r1

### Operating systems (OS)

- [ ] Windows
- [ ] macOS
- [X] Linux or Unix
- [ ] Other

### OS versions or Linux distros

Ubuntu 24.04 armhf

### Synergy configuration

n/a. It's a build issue

### What steps will reproduce the problem?

Attempt to build the product for Ubuntu 24.04 armhf architecture. 
You can find the complete build log here: https://launchpadlibrarian.net/746810820/buildlog_ubuntu-noble-armhf.synergy_1.15.1+r1+dfsg-1xtradeb1.2404.1_BUILDING.txt.gz

The relevant log lines are below:

### Log output

```shell
/<<PKGBUILDDIR>>/src/lib/gui/ipc/QIpcClient.cpp:148:44: error: format ‘%lld’ expects argument of type ‘long long int’, but argument 3 has type ‘qsizetype’ {aka ‘int’} [-Werror=format=]
  148 |     qFatal("unexpected int buffer size: %lld", lenBuf.size());
      |                                         ~~~^   ~~~~~~~~~~~~~
      |                                            |              |
      |                                            long long int  qsizetype {aka int}
      |                                         %d
```

**Comment by sithlord48** [comment]
i would say we can close this we don't support armhf and newer arm aarch64 seams to work @nbolton  Thoughts ?

**Comment by nbolton** [comment]
Yeah, we should not to support ARMHF (32-bit) since it's impractical to do CI with. E.g. in Synergy's early days of Raspberry Pi support we had Raspberry Pi GitHub runners and they took 30 mins to build the source (as opposed to like 1 or 2 mins). You can emulate 32-bit ARM with QEMU but it's also very slow. Since most hardware manufacturers have already moved to 64-bit ARM (e.g. latest Raspberry Pi's), supporting ARMHF (32-bit) seems a bit pointless.

Unless there's some important modern 32-bit use case-case I'm missing...
